tests/tests/regression/issue_6827.rs

```diff
@@ -0,0 +1,301 @@
+use std::sync::Arc;
+
+use wgpu_test::{gpu_test, FailureCase, GpuTestConfiguration, TestParameters, TestingContext};
+
+#[gpu_test]
+static TEST_SINGLE_WRITE: GpuTestConfiguration = GpuTestConfiguration::new()
+    .parameters(
+        TestParameters::default()
+            // See https://github.com/gfx-rs/wgpu/issues/6827
+            .expect_fail(FailureCase::backend_adapter(
+                wgpu::Backends::METAL,
+                "Apple M", // M1,M2 etc
+            )),
+    )
+    .run_async(|ctx| async move { run_test(ctx, true).await });
```

Comment:
Typo causing this test to be identical to TEST_SCATTER:

```suggestion
        TestParameters::default(),
    )
    .run_async(|ctx| async move { run_test(ctx, false).await });
```

Reply:
Bleh, thanks!